crawl_tool/crawl.py

```diff
@@ -0,0 +1,158 @@
+"""Crawl publications from ECMWF homepage."""
+
+import os
+import re
+import time
+import urllib.request
+
+import click
+from bs4 import BeautifulSoup
+
+OUTPUT_ROOT = "/home/user/large-disk/crawled_resources/"
```

Comment:
`output_root` should be set as an argument

Reply:
I think we run this script one time and mainly on our cloud instance and files will be stored at that address. But yes output_folder option argument has been added.